Backend/src/controllers/user.controller.ts

```diff
@@ -147,14 +156,23 @@ export const deposit = async (req: Request, res: Response) => {
       depositAmount: amountInLamports,
     });
 
-    const depositItem: typeof depositAccount.$inferInsert = {
-      deposit_amount: amountInLamports,
-      duration_days,
-      content_cid: computedCID,
-      deposit_key: publicKey.toLowerCase(),
-      deposit_slot: 1,
-      last_claimed_slot: 1,
-    };
+    const client = await initStorachaClient();
+    const cid = await client.uploadFile(files[0]);
```

Comment:
@Devanshgoel-123, kindly revert this... 

storacha client should not be init in the `deposit` function. `uploadFile` is the only place we should call the storacha fns